crates/bevy_gizmos/Cargo.toml

```diff
@@ -9,8 +9,10 @@ license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
 
 [features]
+default = ["fixed_time"]
 webgl = []
 webgpu = []
+fixed_time = ["bevy_app/fixed_time"]
```

Comment:
maybe there should be a comment here or in the `lib.rs` documentation that this crate / plugin doesn't do much without a fixed timestep?

Reply:
Not entirely sure about that? What particular functionality relies on fixed time step? I Moved everything to the regular schedules and it seemed to render fine? maybe I'm missing something

Reply:
Alright, then I was probably mistaken :)